feature/google/src/test/java/com/revenuecat/purchases/google/StoreProductTest.kt

```diff
@@ -1,94 +1,29 @@
-//            priceCurrencyCode = "USD",
-//            originalPrice = "$2",
-//            originalPriceAmountMicros = 2 * 1_000_000,
-//            title = "TITLE",
-//            description = "DESCRIPTION",
-//            subscriptionPeriod = "P1M",
-//            freeTrialPeriod = "P1M",
-//            introductoryPrice = "$0",
-//            introductoryPriceAmountMicros = 0 * 1_000_000,
-//            introductoryPricePeriod = "P1W",
-//            introductoryPriceCycles = 1,
-//            iconUrl = "http://",
-//            originalJson = JSONObject("{}")
-//        )
-//        Assertions.assertThat(storeProduct1).isEqualTo(storeProduct2)
-//    }
-//
-//    @Test
-//    fun `Two StoreProducts with the same properties have the same hashcode`() {
-//        val storeProduct1 = GoogleStoreProduct(
-//            sku = "sku",
-//            type = ProductType.INAPP,
-//            price = "$2",
-//            priceAmountMicros = 2 * 1_000_000,
-//            priceCurrencyCode = "USD",
-//            originalPrice = "$2",
-//            originalPriceAmountMicros = 2 * 1_000_000,
-//            title = "TITLE",
-//            description = "DESCRIPTION",
-//            subscriptionPeriod = "P1M",
-//            freeTrialPeriod = "P1M",
-//            introductoryPrice = "$0",
-//            introductoryPriceAmountMicros = 0 * 1_000_000,
-//            introductoryPricePeriod = "P1W",
-//            introductoryPriceCycles = 1,
-//            iconUrl = "http://",
-//            originalJson = JSONObject("{}")
-//        )
-//        val storeProduct2 = GoogleStoreProduct(
-//            sku = "sku",
-//            type = ProductType.INAPP,
-//            price = "$2",
-//            priceAmountMicros = 2 * 1_000_000,
-//            priceCurrencyCode = "USD",
-//            originalPrice = "$2",
-//            originalPriceAmountMicros = 2 * 1_000_000,
-//            title = "TITLE",
-//            description = "DESCRIPTION",
-//            subscriptionPeriod = "P1M",
-//            freeTrialPeriod = "P1M",
-//            introductoryPrice = "$0",
-//            introductoryPriceAmountMicros = 0 * 1_000_000,
-//            introductoryPricePeriod = "P1W",
-//            introductoryPriceCycles = 1,
-//            iconUrl = "http://",
-//            originalJson = JSONObject("{}")
-//        )
-//        Assertions.assertThat(storeProduct1.hashCode()).isEqualTo(storeProduct2.hashCode())
-//    }
-//}
+@RunWith(AndroidJUnit4::class)
+class StoreProductTest {
+    @Test
+    fun `list of INAPP ProductDetails maps to StoreProducts`() {
+        val productDetail1 = mockProductDetails(productId = "iap_1", type = BillingClient.ProductType.INAPP, subscriptionOfferDetails = null)
+        val productDetails = listOf(productDetail1)
+
+        val storeProducts = productDetails.toStoreProducts()
+        assertThat(storeProducts.size).isEqualTo(1)
+    }
+
+    @Test
+    fun `list of SUBS ProductDetails maps to StoreProducts`() {
```

Comment:
could we add a test to confirm that a list with both maps correctly? 

we also have some other `toStoreProduct` logic we should probably test...i don't remember if we had tests elsewhere for those... (also i know that's not part of this ticket, just wanted to capture somewhere)

Reply:
Added two more tests! 💪